package.json

```diff
@@ -67,6 +67,10 @@
                     "deprecationMessage": "The current profile is now stored internally by the Toolkit.",
                     "description": "%AWS.configuration.profileDescription%"
                 },
+                "aws.auth.ssoCacheDirectory": {
```

Comment:
I saw that one user asked for this. It makes testing easier so I added it.

Reply:
Dev-only `aws.dev` ?